metrics/seqeval/seqeval.py

```diff
@@ -62,80 +75,6 @@
 """
```

Comment:
Could you update the docstring with the new outputs ?

Reply:
Thank you for the review. I updated the docstring.